plugins/romloader/romloader.cpp

```diff
@@ -1553,18 +1554,22 @@ void romloader::cmd_usip(SWIGLUA_REF tLuaFn, long lCallbackUserData)
 
 					/* Get the packet type. */
 					ucPacketTyp = ptPacketHeader->s.ucPacketType;
+					ucPacketSequenceNumber =  ptPacketHeader->s.ucSequenceNumber;
 
 					if( ucPacketTyp==MONITOR_PACKET_TYP_CallMessage )
```

Comment:
I don't remember how exactly the USIP command is handled by the netx, but doesn't it trigger a reset? In that case, there would be no more messages and the romloader method should work more like call_no_answer.